ui-components/src/page_components/listing/UnitTables.tsx

```diff
@@ -78,6 +78,15 @@ const UnitTables = (props: UnitTablesProps) => {
           })
         })
 
+        let areaRangeSection
+        if (unitSummary.areaRange?.min || unitSummary.areaRange?.max) {
+          areaRangeSection = (
+            <>
+              ,&nbsp;{formatRange(unitSummary.areaRange)} {t("t.squareFeet")}
```

Comment:
Can we use template literals to avoid the `,&nbsp;` here? (` ${formatRange...} ${t(...)}`

Reply:
Done.